### PR TITLE
Time Cop v1.7.0

### PR DESCRIPTION
**Pull request by 12people**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/hamaluik/timecop/pull/55#issuecomment-1020440009
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
(The main developer behind the project owns the domain. I have his permission to create the official Flatpak.)
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: ca.hamaluik.Timecop.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+app-id: ca.hamaluik.Timecop
```
Comment (hfiguiere) [inline ca.hamaluik.Timecop.yml:1]: Given that the app website is `timecop.app`, I was wondering if that was the best choice.

File: flathub.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: I would inquire as to why you excluded aarch64 users (RaspberryPi. Pine 64, etc) but then I saw the mention of Flutter upstream.

**Comment by hfiguiere** [comment]
bot, build ca.hamaluik.Timecop

File: ca.hamaluik.Timecop.yml (hunk 1 above)
Comment (hamaluik) [reply ca.hamaluik.Timecop.yml:1]: I built the app before I registered the domain, unfortunately..

**Pull request closed without merging**